pyqtorch/adjoint.py

```diff
@@ -50,14 +51,20 @@ def forward(
         circuit: QuantumCircuit,
         state: Tensor,
         observable: Observable,
+        embedding: Embedding | None,
         param_names: list[str],
         *param_values: Tensor,
     ) -> Tensor:
         ctx.circuit = circuit
         ctx.observable = observable
+        ctx.embedding = embedding
         ctx.param_names = param_names
         values = param_dict(param_names, param_values)
-        ctx.out_state = circuit.run(state, values)
+        if embedding is not None:
+            msg = "AdjointExpectation does not support Embedding yet."
```

Comment:
I am not sure I understand it clearly.

Reply:
the embedding is something which will be used with time-dependent solvers for now whcih are not supported (yet) by adjoint

Reply:
updated